### ISSUE TITLE
Add note from Obashi Meeting

### ISSUE THREAD

**Issue by twsswt** [twsswt/bug_buddy_jira_plugin#14]

**Comment by stephen-brown-socs** [comment]
I met with Paul from OBASHI today to discuss requirements for the project, and how OBASHI could fit into it

I clarified the initial goals of the project, and that we were still in the process of researching recommender systems. Paul asked if we would be taking the history of issues into account, as it would be nice if the system could learn from it. 

Paul clarified that the relation between this project and the OBASHI methodology is that ‘issues’ in a tracker can be easily considered ‘failing components’ in an OBASHI diagram.

I stated that the project will almost certainly be written in Java, as it is what Jira uses. OBASHI is likely to make use of Go, moving forward, and they may want to do a rewrite if the IP is transferred to them.
